cmd/monaco/download/download_entities.go

```diff
@@ -108,15 +105,15 @@ func (d DefaultCommand) DownloadEntities(fs afero.Fs, cmdOptions entitiesDirectD
 					Value: token,
 				},
 			},
-			outputFolder:            cmdOptions.outputFolder,
-			projectName:             cmdOptions.projectName,
-			forceOverwriteManifest:  cmdOptions.forceOverwrite,
-			concurrentDownloadLimit: concurrentDownloadLimit,
+			outputFolder:           cmdOptions.outputFolder,
+			projectName:            cmdOptions.projectName,
+			forceOverwriteManifest: cmdOptions.forceOverwrite,
 		},
 		specificEntitiesTypes: cmdOptions.specificEntitiesTypes,
 	}
 
-	dtClient, err := client.NewClassicClient(cmdOptions.environmentUrl, token)
+	dtClient, err := client.NewClassicClient(cmdOptions.environmentUrl,
+		token, client.WithClientRequestLimiter(concurrency.NewLimiter(environment.GetEnvValueIntLog(environment.ConcurrentRequestsEnvKey))))
```

Comment:
Should we just hide the limiter somewhere inside instead of exposing it to consumers?

Reply:
I agree with @Laubi - I don't think there's any case where we don't use the limiter anymore, so like the version check we should see it as a default (maybe introduce "default options"?) so we don't forget about it the next time we construct a client?

Reply:
You are probably right - I will change the code so that the client uses a limiter with its values read from the environment variable in any case by default.

Also, I was thinking about not exposing the limiter at all, But then I had the feeling that this component is basically designed for reuse (from any differnt package).  However, I just see that It is not so obvious to reuse it, e.g.:

```
type MyFancyComponent struct {
 limiter concurrency.Limiter
}
```

will panic as Limiter does not provide usable zero values, i.e. you need to remember to construct it with `NewLimiter`.
This can be fixed with guards of course.